Source/WebKit/WebProcess/WebPage/WebBackForwardListProxy.cpp

```diff
@@ -58,15 +58,28 @@ WebBackForwardListProxy::WebBackForwardListProxy(WebPage& page)
 {
 }
 
+static const unsigned DefaultCapacity = 100; // afryer: TODO: this should use the same const (which will improve readability).
```

Comment:
This could probably live in `WebCore/history/BackForwardItemIdentifier.h` as a constexpr.